packages/gatsby/cache-dir/query-engine/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// IGatsbyPage, SystemPath and copied/inlined from redux/types so this file is self contained
+type SystemPath = string
+type Identifier = string
+
+export interface IGatsbyPage {
+  internalComponentName: string
+  path: string
+  matchPath: undefined | string
+  component: SystemPath
+  componentChunkName: string
+  isCreatedByStatefulCreatePages: boolean
+  context: Record<string, unknown>
+  updatedAt: number
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  pluginCreator___NODE: Identifier
+  pluginCreatorId: Identifier
+  componentPath: SystemPath
+  ownerNodeId: Identifier
+}
```

Comment:
Ultimately we don't need those `index.d.ts` files at all - those were only meant to provide type information when utilizing built engine and because that can be used outside of site directory later I had to make it to not rely on any type imports.